tox.ini

```diff
@@ -1,6 +1,13 @@
 [tox]
 envlist = py27
 
+[pylama:pycodestyle]
+max_line_length = 100
+
+[pylama:pylint]
+max_line_length = 100
+disable = R
+
```

Comment:
Lets not put those configurations into the tox.ini bug use setup.cfg for it. See https://pylama.readthedocs.io/en/latest/#configuration-files.

Also I would prefer to have a general pylama section and not specific ones for each tool used. Does `[pylama]` not work? Also what is `disable = R`? Is that really something we need?